spring-rabbit/src/test/java/org/springframework/amqp/rabbit/listener/SimpleMessageListenerContainerTests.java

```diff
@@ -784,4 +809,33 @@ protected void doRollback(DefaultTransactionStatus status) throws TransactionExc
 
 	}
 
+	@SuppressWarnings("serial")
+	private class TestExecutor extends SimpleAsyncTaskExecutor {
```

Comment:
```suggestion
	private static final class TestExecutor extends SimpleAsyncTaskExecutor {
```

Also, please, add your name to the `@author` list of affected classes.